css/style.css

```diff
@@ -0,0 +1,462 @@
+.container {
+  background: url(../img/Background.png) no-repeat;
+  background-size: cover;
+  display: flex;
+  max-width: 1440px;
+  min-width: 320px;
+  height: 100vh;
+  margin: auto;
+}
+
+.menu { margin: 0 0 0 100px; }
+
+.title {
+  color: #fff;
+  margin: 50px 0 20px 0;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 50px;
+  line-height: 59px;
+}
+
+.subtitle {
+  color: #fff;
+  margin: 0 0 60px 0;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 19px;
+  line-height: 22px;
+}
+
+.menu__select-level { position: relative; }
+
+.level_btn {
+  display: flex;
+  margin-bottom: 20px;
+  position: relative;
+  background: none;
+  border: none;
+  color: #fff;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 50px;
+  line-height: 59px;
+  outline: none;
+  padding-left: 40px;
+}
+
+.level_btn:hover {
+  opacity: .8;
+  cursor: pointer;
+  position: relative;
+}
+
+.level_active::before {
+  content: '';
+  background-color: #d62b2e;
+  position: absolute;
+  top: 20px;
+  left: 0;
+  width: 20px;
+  height: 20px;
+  transform: rotate(45deg);
+}
+
+.start-button {
+  background: #d62b2e;
+  border: none;
+  border-radius: 10px;
+  color: #fff;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 50px;
+  height: 90px;
+  width: 350px;
+  outline: none; 
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Olga_FinalGame_JS/pullRequest?prid=5839198)